python/setup.py

```diff
@@ -2,7 +2,7 @@
 
 setup(
     name='Golem-Task-Api',
-    version='0.24.1',
+    version='0.24.2',
```

Comment:
just to check: No interface changes because the new param is optional?

Reply:
Yes - the code written for 0.24.1 will work with 0.24.2 without changes